src/styles/blocks/card.scss

```
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
```

since we'll add padding to a card
```suggestion
    display: block;
    margin: 0 auto 40px;
```